x/auth/types/codec.go

```diff
@@ -2,32 +2,31 @@ package types
 
 import (
 	"github.com/cosmos/cosmos-sdk/codec"
-	"github.com/cosmos/cosmos-sdk/x/auth/exported"
+	"github.com/cosmos/cosmos-sdk/codec/types"
 )
 
-// Codec defines the interface needed to serialize x/auth state. It must be
-// aware of all concrete account types.
-type Codec interface {
-	codec.Marshaler
-
-	MarshalAccount(acc exported.Account) ([]byte, error)
-	UnmarshalAccount(bz []byte) (exported.Account, error)
-
-	MarshalAccountJSON(acc exported.Account) ([]byte, error)
-	UnmarshalAccountJSON(bz []byte) (exported.Account, error)
-}
-
 // RegisterCodec registers the account interfaces and concrete types on the
 // provided Amino codec.
 func RegisterCodec(cdc *codec.Codec) {
-	cdc.RegisterInterface((*exported.ModuleAccountI)(nil), nil)
-	cdc.RegisterInterface((*exported.GenesisAccount)(nil), nil)
-	cdc.RegisterInterface((*exported.Account)(nil), nil)
-	cdc.RegisterConcrete(&BaseAccount{}, "cosmos-sdk/Account", nil)
+	cdc.RegisterInterface((*ModuleAccountI)(nil), nil)
+	cdc.RegisterInterface((*GenesisAccount)(nil), nil)
+	cdc.RegisterInterface((*AccountI)(nil), nil)
+	cdc.RegisterConcrete(&BaseAccount{}, "cosmos-sdk/AccountI", nil)
```

Comment:
```suggestion
	cdc.RegisterConcrete(&BaseAccount{}, "cosmos-sdk/Account", nil)
```

Reply:
I think this should also be "cosmos-sdk/Baseaccount" as suggested by bez in legacy https://github.com/cosmos/cosmos-sdk/pull/6165#discussion_r423307878

Reply:
Okay, let's do that then